src/core/services-impl/pom.xml

```diff
@@ -171,30 +171,39 @@
     </build>
 
     <profiles>
+        <!-- coverage with JaCoCo instead of Cobertura -->
```

Comment:
Can you please briefly report this in PR description with a more coherent PR title? Thanks.